css/styles.css

```diff
@@ -53,6 +54,15 @@ So I grouped all my buttons here
   width: max-content;
 }
 
+button {
+  background-color: var(--my-project-boutton-color);
+  color: white;
+  padding: 3% 3%;
+  margin: 3% auto 10% auto;
+  width: max-content;
+  border-radius: 4%;
+}
```

Comment:
- [ ] Please I kindly suggest adding button effects (enable, hover, pressed, etc.) per project requirements. 😄 

![Screenshot (121)](https://user-images.githubusercontent.com/78104544/228727560-715337cb-1834-4afa-86f9-b404a25a4372.png)

Please this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) is provided for you to cross-check the project requirements. 😅

Reply:
Done @braincee Thanks